lib/misc/cells.js

```diff
@@ -0,0 +1,64 @@
+(function() {
+  var scopes, values;
+
+  scopes = require('./scopes');
+
+  values = require('underscore-plus').values;
```

Comment:
Probably need `'use babel'` at the top of the file, like e.g. [here](https://github.com/JunoLab/atom-ink/blob/b3b9d15ae6da254bc2bc04c1449500db98520a63/lib/plots/pane.js).